src/Logging/Logging.EventSource/test/EventSourceLoggerTest.cs

```diff
@@ -694,13 +694,13 @@ private static class EventTypes
 #if NETCOREAPP
             { "E5JS", (e) => VerifySingleEvent(e, "Logger2", EventTypes.MessageJson, 5, null, LogLevel.Critical,
                 @"""ArgumentsJson"":{""stringParam"":""bar"",""int1Param"":""23"",""int2Param"":""45""",
-                @$"""ExceptionJson"":{{""TypeName"":""System.Exception"",""Message"":""oops"",""HResult"":""-2146233088"",""VerboseMessage"":""System.Exception: oops{EscapedNewline()} ---\u003e System.Exception: inner oops") },
+                @$"""ExceptionJson"":{{""TypeName"":""System.Exception"",""Message"":""oops"",""HResult"":""-2146233088"",""VerboseMessage"":""System.Exception: oops{Environment.NewLine} ---\\u003e System.Exception: inner oops") },
```

Comment:
@ahsonkhan and @steveharter 
Do these changes look correct based on recent changes?

Reply:
I don't know about `EscapedNewline()` -> `Environment.NewLine` (but that seems to have been reverted).

Also, I don't know why the change to add an extra `\` to `\\u003e` was made.

Looks like these changes are old. What's currently in the PR with latest commits (changing `e` to `E`) is correct.

Reply:
Yeah, difference between my local build and CI build, ignore this commit